src/chains/ethereum/transaction/src/transaction-factory.ts

```diff
@@ -117,8 +117,21 @@ export class TransactionFactory {
             extra
           );
           if (toEIP1559) {
-            tx.maxFeePerGas = Quantity.from(null);
-            tx.maxPriorityFeePerGas = RPCQUANTITY_ZERO;
+            // they didn't specify the type as eip-1559 (type 2), so we are
+            // upgrading it. BUT, there's still a chance they sent us this data,
+            // so we don't want to overwrite it.
+            if (
+              txData.maxFeePerGas === undefined ||
+              txData.maxFeePerGas === ""
+            ) {
+              tx.maxFeePerGas = Quantity.from(null);
+            }
+            if (
+              txData.maxPriorityFeePerGas === undefined ||
+              txData.maxPriorityFeePerGas === ""
+            ) {
+              tx.maxPriorityFeePerGas = RPCQUANTITY_ZERO;
```

Comment:
Let's just do a falsey check on these e because  the empty string case should be impossible anyway, as these two tx.* values are Quantities, which won't allow `""` anyway.